tests/console/snapper_create.pm

```diff
@@ -15,11 +15,21 @@ use base 'btrfs_test';
 use testapi;
 use utils;
 
+sub get_last_snap_number {
+    # In many cases script output returns not only script execution results
+    # but other data which was written to serial device. We have to ensure
+    # that we got what we expect. See poo#25716
+    my $output = script_output("snapper list | tail -n1 | awk '{ print \">>>\" \$3 \"<<<\" }'");
```

Comment:
Wasn't there a ticket for os-autoinst which should handle it there instead of coming up with something special in tests?

Reply:
Yes, there should be one, but I was not able to find it. Shall I add soft failure then? Unfortunately, I don't expect it to be fixed any time soon.

Reply:
no soft fail but please ensure it's tracked in os-autoinst, *at least* good documentation for the method